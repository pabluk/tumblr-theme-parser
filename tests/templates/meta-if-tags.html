<html>
    <head>
        <meta name="if:Show people I follow" content="1"/>
        <meta name="if:Reverse pagination" content="0"/>
    </head>
    <body>
        {block:IfNotReversePagination}
            <a href="...">Previous</a> <a href="...">Next</a>
        {/block:IfNotReversePagination}

        {block:IfReversePagination}
            <a href="...">Next</a> <a href="...">Previous</a>
        {/block:IfReversePagination}

        {block:IfShowPeopleIFollow}
            <div id="following">...</div>
        {/block:IfShowPeopleIFollow}
    </body>
</html>
