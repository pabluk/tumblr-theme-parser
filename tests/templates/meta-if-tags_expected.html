<html>
    <head>
        <meta name="if:Show people I follow" content="1"/>
        <meta name="if:Reverse pagination" content="0"/>
    </head>
    <body>
        <a href="...">Previous</a> <a href="...">Next</a>
        

        

        <div id="following">...</div>
        
    </body>
</html>
